src/subsystem/nested_subsystem.rs

```diff
@@ -83,11 +83,24 @@ impl<ErrType: ErrTypeTraits> NestedSubsystem<ErrType> {
         self.error_actions.on_panic.store(action, Ordering::Relaxed);
     }
 
-    /// Returns a future that resolves once the subsystem is finished.
+    /// Returns a future that resolves once the subsystem and its children are finished.
     ///
     /// Similar to [`join`](NestedSubsystem::join), but more light-weight
     /// as it does not return any information about subsystem errors.
     pub fn finished(&self) -> SubsystemFinishedFuture {
         SubsystemFinishedFuture::new(self.joiner.clone())
     }
+
+    /// Returns whether the subsystem or any of its children are running.
```

Comment:
`Returns whether the subsytem and all of its children are finished.`, to be consistent with `finished()`.